service/src/main/java/bio/terra/landingzone/db/LandingZoneDao.java

```diff
@@ -0,0 +1,152 @@
+package bio.terra.landingzone.db;
+
+import bio.terra.common.db.ReadTransaction;
+import bio.terra.common.db.WriteTransaction;
+import bio.terra.common.exception.MissingRequiredFieldException;
+import bio.terra.landingzone.db.exception.DuplicateLandingZoneException;
+import bio.terra.landingzone.db.exception.LandingZoneNotFoundException;
+import bio.terra.landingzone.sevice.model.LandingZone;
+import bio.terra.workspace.db.DbSerDes; // TODO move this utility to terra common.db.
+import java.util.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.dao.support.DataAccessUtils;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Component;
+
+/** LandingZoneDao includes operations on the landing zone tables. */
+@Component
+public class LandingZoneDao {
+  /** SQL query for reading landing zone records. */
+  private static final String LANDINGZONE_SELECT_SQL =
+      "SELECT landingzone_id, spend_profile, template_id, template_version, properties"
```

Comment:
Maybe definition is better than template? You proposed it initially. And for landing zone listing rest api we are using path you've suggested - /landingzones/definitions.. So, I am thinking maybe it's better to use definition here as well..